__device-tests__/helpers/caps.js

```diff
@@ -0,0 +1,25 @@
+exports.ios12 = {
+	browserName: '',
+	platformName: 'iOS',
+	platformVersion: '12.0',
+	deviceName: 'iPhone Simulator',
+	os: 'iOS',
+	deviceOrientation: 'portrait',
+	automationName: 'XCUITest',
+	appiumVersion: '1.9.1',
```

Comment:
Not something that we need to address in this PR but, I wonder if we can have a way to match the version number here with the one in `package.json` so they are kept in sync, or maybe some warning raised if they don't match.

Reply:
Right, I've noted this... The need for the Appium version being specified is actually a result of the requirement of SauceLabs, otherwise there'd be no need and would just pick it up automatically.

Reply:
> the Appium version being specified is actually a result of the requirement of SauceLabs

Oh, interesting. Let's add a line comment about this, it's quite a requirement apparently.

Reply:
Seems addressed by 8f6b24ef43eb613ff300b0ad228411e3fc40041a.